ui/src/app/cohort-review/create-review-modal/create-review-modal.tsx

```diff
@@ -76,9 +77,8 @@ export const CreateReviewModal = withCurrentWorkspace()(
       };
     }
 
-    cancelReview() {
```

Comment:
(No action required) These deletion seems interesting – is this just a drive-by cleanup?

Reply:
Yep, this was an unused function that should have been removed previously.